src/cookbook.js

```diff
@@ -4,13 +4,19 @@ class Cookbook {
   }
 
   findRecipe(searchText) {
+    const newSearchText = searchText.toLowerCase()
     return this.recipes.filter(recipe => {
       return recipe.ingredients.find(ingredient => {
-        return (ingredient.name.includes(searchText)) ||
-        (recipe.name.includes(searchText))
+        return (ingredient.name.toLowerCase().includes(newSearchText)) ||
+       (recipe.name.toLowerCase().includes(newSearchText))
       });
     })
   }
+
+  findRecipeByTags(tag) {
+    return this.recipes.filter(recipe => recipe.tags.includes(tag))
```

Comment:
I've add the tag vs tags issue to our project board - something we can brainstorm!